app/src/main/java/com/bignerdranch/android/geoquiz/MainActivity.kt

```diff
@@ -62,7 +74,14 @@ class MainActivity : AppCompatActivity() {
         } else {
             R.string.incorrect_toast
         }
+
+
         Toast.makeText(this, messageResId, Toast.LENGTH_SHORT)
             .show()
+
+        var myToast : Toast= Toast.makeText(this, messageResId, Toast.LENGTH_SHORT)
```

Comment:
Nice work but don't need to duplicate Toast.
I recommend removing lines (79, 80)